### DISCUSSION TITLE
Server is not starting need help

### DISCUSSION THREAD

**Discussion by SnackyWell** [PR3SIDENT/enshrouded-server#22] [Q&A] [answered]
Hi @DrSh4d0w ,

I encountered an issue after running the ./StartEnshroudedServer.sh script. The server does not start, and I suspect it might be related to permission settings, but I'm not sure.

Here's a link to the output I received: [output.txt](https://github.com/PR3SIDENT/enshrouded-server/files/14043549/output.txt)

One potential error that stood out to me is:
`[resource] Could not load resource cache file 'Z:/home/steam/enshrouded/enshrouded_server.krc'. error=file not found`

For reference, I used the latest installer available. Any assistance or insights into this issue would be greatly appreciated.

**Comment by VincentRoma** [comment]
Same issue here. 
Ubuntu 22.04, Wine 9, tried Docker & Direct run with wine.

**Reply by SnackyWell** [reply]
Same actulaly.

**Reply by fhazal** [reply]
i have this issue when running on ubuntu 22.04 

```
The following packages have unmet dependencies:
 winehq-staging : Depends: wine-staging (= 9.0.0~jammy-1)
E: Unable to correct problems, you have held broken packages.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
screen is already the newest version (4.9.0-1).
screen set to manually installed.

```
```
dlmopen steamservice.so failed: steamservice.so: cannot open shared object file: No such file or directory
dlmopen steamservice.so failed: steamservice.so: cannot open shared object file: No such file or directory
Success! App '2278520' fully installed.
CWorkThreadPool::~CWorkThreadPool: work complete queue not empty, 1 items discarded.
```
```
enshrouded@HPZ440-Server:~# sudo systemctl stop enshrouded.service
enshrouded@HPZ440-Server:~# sudo systemctl status enshrouded.service
○ enshrouded.service - Enshrouded Server
     Loaded: loaded (/etc/systemd/system/enshrouded.service; enabled; vendor preset: enabled)
     Active: inactive (dead) (Result: exit-code) since Thu 2024-01-25 22:59:54 +08; 2s ago
    Process: 487768 ExecStartPre=/usr/games/steamcmd +@sSteamCmdForcePlatformType windows +force_install_dir /home/steam/enshrouded +login anonymous +app_update 2278520 +quit (code=exited, status=0/SUCCESS)
    Process: 487796 ExecStart=/home/steam/enshrouded/StartEnshroudedServer.sh (code=exited, status=127)
   Main PID: 487796 (code=exited, status=127)
        CPU: 1.296s

```

**Reply by VincentRoma** [reply, marked as answer]
If you're still stuck, try this one, it's working fine first try: https://github.com/mornedhels/enshrouded-server

**Comment by ThrashVTX** [comment]
If you're deploying to a guest vm  on a hypervisor, try selecting 'host' as the guest vm's cpu type. Tested on kvm. (Proxmox) Other virtualized instruction sets may work as well, those were not tested. The underlying physical CPU in my case is an Intel Core i9-13900.
